Packages/com.unity.inputsystem/InputSystem/Actions/InputActionProperty.cs

```diff
@@ -33,12 +33,12 @@ public struct InputActionProperty : IEquatable<InputActionProperty>, IEquatable<
         /// </summary>
         /// <value>The action object contained in the property.</value>
         /// <remarks>
-        /// This property will return <c>null</c> if the property using a <see cref="reference"/> and
+        /// This property will return <c>null</c> if the property is using a <see cref="reference"/> and
         /// the referenced action cannot be found. Also, it will be <c>null</c> if the property
         /// has been manually initialized with a <c>null</c> <see cref="InputAction"/> using
         /// <see cref="InputActionProperty(InputAction)"/>.
         /// </remarks>
-        public InputAction action => m_UseReference ? m_Reference.action : m_Action;
+        public InputAction action => m_UseReference ? m_Reference != null ? m_Reference.action : null : m_Action;
```

Comment:
`m_UseReference ? m_Reference?.action : m_Action`

Reply:
I would have used `?.` but that wouldn't use the overloaded `operator==` of `UnityEngine.Object` which also checks if the object is valid.